definitions/cross_product.toml

```
@@ -0,0 +1,53 @@
+[metrics]
+
+[metrics.mobile_daily_active_users]
```

For newly added metrics it is recommended to version them. Just to make it easier to deal with future changes

```suggestion
[metrics.mobile_daily_active_users_v1]
```